index.css

```diff
@@ -0,0 +1,107 @@
+#add-book,
+#contact-us {
+  display: none;
+}
+
+.book-list-container {
+  border: 2px black solid;
+  display: flex;
+  justify-content: space-between;
+  padding: 0.25rem 1rem;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 10px;
+}
+
+h3 {
+  margin: 0;
+}
+
+a {
+  padding: 0.25rem 0.5rem;
+}
+
+.vertical-hr {
+  border-left: 2px solid black;
+}
+
+#date-time {
+  float: right;
+  margin-top: 5px;
+}
+
+#all-books h1 {
+  text-align: center;
+}
+
+#book-list {
+  border: black 4px solid;
+  padding: 0;
+}
+
+#all-books ul {
+  list-style-type: none;
+}
+
+#all-books ul li {
+  padding: 0 0.5rem;
+}
+
+#all-books ul li:nth-of-type(odd) {
+  background-color: lightgray;
+}
+
+hr {
+  margin-bottom: 0;
+}
+
+#remove-button {
+  float: right;
+  color: black;
+  background-color: white;
+  box-shadow: 0 0 5px rgba(0, 0, 0, 1);
+  border: 4px solid black;
+}
+
+#add-book-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#add-book-form h2 {
+  margin-bottom: 0;
+}
+
+input {
+  margin: 0.5rem;
+  padding: 0.5rem;
+  border: 3px solid black;
+  min-width: 300px;
+}
+
+#add-button {
+  margin: 0.5rem;
+  padding: 0.2rem 0.5rem;
+  border: 4px solid black;
+  background-color: white;
+  color: black;
+  box-shadow: 0 0 5px rgba(0, 0, 0, 1);
+  margin-left: 17.8125rem;
+}
+
+.show {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+footer {
+  border: 2px black solid;
+  padding-left: 20px;
+}
```

Comment:
Your footer is well-styled and looks good 👍 However, It would be great if it remained fixed at the bottom of the page as conventional footers on webpages 😉 This would improve the user experience on your page. 
How it looks at present: 
![image](https://user-images.githubusercontent.com/107795045/229943067-36d8f9fa-94d2-46e4-89aa-3474c2c01cab.png)